src/vs/base/parts/quickinput/common/quickInput.ts

```diff
@@ -353,26 +353,61 @@ export interface IQuickInputToggle {
 
 export interface IInputBox extends IQuickInput {
 
+	/**
+	 * Initial value shown in the input box.
```

Comment:
```suggestion
	 * Value shown in the input box.
```
I removed initial because you can set the value after the input box is shown